modules/ccs-gcp-customer-procedure.adoc

```diff
@@ -29,53 +29,67 @@ To use {product-title} in your GCP project, the following GCP organizational pol
+// |Purpose
 
-|link:https://console.cloud.google.com/apis/library/iamcredentials.googleapis.com[IAM Service Account Credentials API]
+|link:https://cloud.google.com/iam/docs/reference/credentials/rest[IAM Service Account Credentials API]
 |`iamcredentials.googleapis.com`
+|Used for creating short-lived credentials for impersonating IAM service accounts.
 
-|link:https://console.cloud.google.com/apis/library/iam.googleapis.com?project=openshift-gce-devel&folder=&organizationId=[Identity and Access Management (IAM) API]
+|link:https://cloud.google.com/iam/docs/reference/rest[Identity and Access Management (IAM) API]
 |`iam.googleapis.com`
+|Used for managing the IAM configuration for the cluster.
 
-|link:https://console.cloud.google.com/apis/library/servicemanagement.googleapis.com?project=openshift-gce-devel&folder=&organizationId=[Service Management API]
+|link:https://cloud.google.com/service-infrastructure/docs/service-management/reference/rest[Service Management API]
 |`servicemanagement.googleapis.com`
+|Used indirectly to fetch quota information for GCP resources.
 
-|link:https://console.cloud.google.com/apis/library/serviceusage.googleapis.com?project=openshift-gce-devel&folder=&organizationId=[Service Usage API]
+|link:https://cloud.google.com/service-usage/docs/reference/rest[Service Usage API]
 |`serviceusage.googleapis.com`
+|Used for determining what services are available in the customer’s Google Cloud account.
 
-|link:https://console.cloud.google.com/apis/library/storage-api.googleapis.com?project=openshift-gce-devel&folder=&organizationId=[Google Cloud Storage JSON API]
+|link:https://cloud.google.com/storage/docs/json_api[Cloud Storage JSON API]
 |`storage-api.googleapis.com`
+|Used for accessing Cloud Storage for the image registry, ignition, and cluster backups (if applicable).
 
-|link:https://console.cloud.google.com/apis/library/storage-component.googleapis.com?project=openshift-gce-devel&folder=&organizationId=[Cloud Storage]
+|link:https://cloud.google.com/storage/docs/apis[Cloud Storage]
 |`storage-component.googleapis.com`
+|Used for managing Cloud Storage for the image registry, ignition, and cluster backups (if applicable).
 
-|link:https://console.cloud.google.com/apis/library/orgpolicy.googleapis.com?project=openshift-gce-devel&folder=&organizationId=[Organization Policy API]
+|link:https://cloud.google.com/resource-manager/docs/reference/orgpolicy/rest[Organization Policy API]
 |`orgpolicy.googleapis.com`
+|Used to identify governance rules applied to customer’s Google Cloud that might impact cluster creation or management.
 
-|link:https://console.cloud.google.com/marketplace/product/google/iap.googleapis.com?q=search&referrer=search&hl=en&project=openshift-gce-devel[Cloud Identity-Aware Proxy API]
+|link:https://cloud.google.com/iap/docs/reference/rest[Cloud Identity-Aware Proxy API]
 |`iap.googleapis.com` ^[*]^
+|Used in emergency situations to troubleshoot cluster nodes that are otherwise inaccessible.
```

Comment:
I see this is existing, but given that `*` could be a character in a URL and this `*` is an asterisk for a footnote, do you think moving it to the Purpose text makes sense? Or, since `[.small]` is technically not supported and we aren't supposed to use this anymore, maybe just bringing the statement into the box entirely?
```suggestion
|link:https://cloud.google.com/iap/docs/reference/rest[Cloud Identity-Aware Proxy API]
|`iap.googleapis.com`
|Used in emergency situations to troubleshoot cluster nodes that are otherwise inaccessible. This API is required for clusters deployed with Private Service Connect.
```

Reply:
Ah, good call, @jeana-redhat! I will move it in to the Purpose text. Thanks!!